sdk/python/kubeflow/storage_initializer/requirements.txt

```diff
@@ -2,7 +2,7 @@ einops>=0.6.1
 transformers_stream_generator==0.0.4
 boto3==1.33.9
 transformers>=4.20.0
-peft>=0.3.0
+peft==0.3.0
```

Comment:
Why do we need to constrain to v0.3.0?

Reply:
so that there is no version conflict in the users virtual environment and the peft inside the container. If there is a mismatch it throws an error